e2e/deploy-vault.go

```diff
@@ -91,3 +97,79 @@ func createORDeleteVault(action string) {
 		e2elog.Failf("failed to %s vault psp %v", action, err)
 	}
 }
+
+// waitForJobCompletion polls the status of the given job and waits until the
+// jobs has succeeded or until the timeout is hit.
+func waitForJobCompletion(c kubernetes.Interface, ns, job string, timeout int) error {
```

Comment:
move this to utils.go or add a new file for job-specific helper functions.

Reply:
moved to utils.go